Index: app/src/main/java/com/sokoloma777/mynewapplication/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/sokoloma777/mynewapplication/MainActivity.java	(revision fbae48617a846d0ce2476b35655ab8896d388b1c)
+++ app/src/main/java/com/sokoloma777/mynewapplication/MainActivity.java	(date 1591893499000)
@@ -54,13 +54,7 @@
                 changeIntToArray(R.array.male_persons_items)));
         category_index = 3;
 
-        /*personsShow(
-                R.array.full_names_of_female_persons,
-                R.array.discriptions_of_female_persons,
-                R.array.image_of_female_persons,
-                R.array.female_persons_items);
 
-         */
 
         myRecyclerAdapter = new MyRecyclerAdapter(myArray, category_index,this);
         rcView.setAdapter(myRecyclerAdapter);
